distribution/openrct2.d.ts

```diff
@@ -4277,11 +4277,11 @@ declare global {
      */
     type WidgetType =
         "button" | "checkbox" | "colourpicker" | "custom" | "dropdown" | "groupbox" |
-        "label" | "listview" | "spinner" | "textbox" | "viewport";
+        "label" | "listview" | "spinner" | "textbox" | "viewport" | "progress_bar";
```

Comment:
This should be alphabetically ordered.
To follow the naming convention, it should be called `progressbar`.